Solutions/DruvaDataSecurityCloud/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"publisherId": "azuresentinel",
```

Comment:
Hello @Nidhi-druva & @shubham-deshmukh-druva, Is this Solution  Microsoft supported ?

We haven't received any instructions regarding a new solution supported by Microsoft.
Please confirm on this or update publisherId and support details accordingly

Reply:
This is not a Microsoft provided Solution.
We have updated the support details and publisher ID. Not able to create new publisher in partner account. But we will use this publisher ID while creation.

Reply:
Update:
Created the publisher ID: druva-azuresentinel-solution